tracer/src/Datadog.Trace/Ci/TestModule.cs

```diff
@@ -211,7 +216,19 @@ internal TestModule(string name, string? framework, string? frameworkVersion, Da
     /// </summary>
     /// <param name="name">Test module name</param>
     /// <returns>New test module instance</returns>
+    [PublicApi]
     public static TestModule Create(string name)
+    {
+        TelemetryFactory.Metrics.RecordCountCIVisibilityManualApiEvent(MetricTags.CIVisibilityTestingEventType.Module);
```

Comment:
I don't have any string opinion about it, if you want to add it we can do it

Reply:
meh, maybe @pierotibou has strong opinions about it 😄 I think it's fine as-is tbh

Reply:
My reflexion:
Pros
- the public api of the library is public cross product. So it could be nice to have everything in one place as if we want to do a new major we can have a look to api usage across products easily. That will still be possible with the current suggestion, just a tad harder

Cons
- I don't think we should have 2 metrics emitted here. So if we are moving to `public_api` namespace, I'd not send the other one (mainly for perf reasons). But that's more allocations on ci app, so I'm not the one it impacts :p
- I think the most important point is to have the same format for metrics across the product. So having the same metrics for all civisibility languages would be my first prio

so ... meh :p, yes no strong opinion and seems fine as is